packages/common-all/src/types/foundation.ts

```diff
@@ -183,6 +183,8 @@ export type NoteSEOProps = {
   updated: number;
   created: number;
   excerpt?: string;
+  image?: string;
```

Comment:
can you add some docstrings describing these properties?

Reply:
Will do

Reply:
Will do

Reply:
Hey @kevinslin i adeed the docString for the new image prop.